### DISCUSSION TITLE
Empty map when using filter ?

### DISCUSSION THREAD

**Discussion by Mateleo** [deckerst/aves#861] [Q&A]
Is it normal that when I use the "no location" filter and try to mark the photo using the map function, no photo is displayed?
But when I try without the filter, I can see the other photos according to their location.

**Comment by deckerst** [comment]
It's normal, but I understand it may be undesired in the specific case of the "no location" filter.

Usually it's useful to apply the filters of the current collection, so you see on the map only related and relevant media.

Did you ask out of curiosity, or would it be useful to you to see other media when the "no location" filter is applied?

**Reply by Mateleo** [reply]
I asked out of curiosity, but for my part it would be super useful. Because sometimes I can't remember the exact location, but having the location of nearby photos is really useful. I use the app mainly to tag photos of missing positions.

**Comment by deckerst** [comment]
Done.
